src/app/item-details/item-details.component.html

```diff
@@ -1,2 +1,49 @@
-<p class="selectItemParagraph">
-    Select an Action Item to view details</p>
\ No newline at end of file
+<div *ngIf="item !== undefined && isLoadingActionItem" class="spinner">
+    <i class="fa fa-spinner fa-spin"></i>
+</div>
+<div *ngIf="item === undefined && errorMessage" class="selectItem">
+    <p>{{errorMessage}}</p>
+</div>
+<div class="details-container" *ngIf="!isLoadingActionItem && item !== undefined">
+    <div class="details" *ngIf="item">
+        <div class="detailsTop">
+            <div class="detail title">
+                <div class="top title">
+                    <div class="itemTitle">{{item.title}}</div>
+                </div>
+                <div class="top">
+                    <div class="detailIcon"><i class="fa fa-edit"></i></div>
+                    <div class="detailIcon"><i class="fa fa-check"></i></div>
+                </div>
+            </div>
+            <div class="detail projectName">
+                <div>{{item.projectName}}</div>
+            </div>
+        </div>
+        <div class="center">
+            <div class="detail dueDate">
+                <div>Due Date: {{(item.dueDate | date: 'MM/dd/yyyy') || '-' }}</div>
+            </div>
+            <div class="circularDiagrams">
+                <div class="diagram positionRelative" [class.diagramsHidden]="!itemVisibility">
```

Comment:
Please use `ngIf` directive instead of adding class with display none attribute.

Reply:
This one is still not addressed.